src/UI/Component/Clickable.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace ILIAS\UI\Component;
```

Comment:
To be discussed: Not sure if I like the path in which those interfaces are stored. I am afraid that we will create a huge mess if we continue to spam the component directory with all sorts of uncategorized interfaces.